src/main/java/com/sourcegraph/find/browser/BrowserAndLoadingPanel.java

```diff
@@ -76,7 +75,7 @@ private void refreshUI() {
 
     } else if (errorMessage != null) {
       String wrappedText = WordUtils.wrap("Error: " + errorMessage, 100);
-      String[] lines = wrappedText.split(SystemUtils.LINE_SEPARATOR);
+      String[] lines = wrappedText.split(System.lineSeparator());
```

Comment:
I remember some troubles with this one. Please check and review the UI in the latest and the oldest supported versions 🙏

Reply:
I'm just going to split by `\n`. This should work on all platforms.

Reply:
> I remember some troubles with this one
There was a problem with a separator but it was for UI separator element - maybe you think about that other one?:)